docs/content/en/docs-dev/user-guide/configuring-deployment/deployment-chain.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Deployment Chain"
+linkTitle: "Deployment Chain"
+weight: 99
+description: >
+  Specific guide for configuring chain of deployments.
+---
+
+For users who want to use PipeCD to build a complex deployment flow, which contains multiple applications across multiple application kinds, this guideline will show you how to use PipeCD to archive that requirement.
+
+## Configuration
+
+The idea of this feature is to trigger the whole deployment chain when a specified deployment is triggered. To enable trigger the deployment chain, we need to add a configuration section named `postSync` which contains all configurations that be used when the deployment is triggered. For this `Deployment Chain` feature, configuration for it is under `postSync.chain` section.
+
+A canonical configuration look as below:
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Kubernetes
+spec:
+  input:
+    ...
+  pipeline:
+    ...
+  postSync:
+    chain:
+      applications:
+        - name: Application 2
+        - name: Application 3
+```
+
+As a result, the above configuration will be used to create a deployment chain like the below figure
+
+![](/images/deployment-chain-figure.png)
```

Comment:
I guess the reader will be a bit confused at first when looking at this diagram since there are many `Application 2`.
How about using `Label` to filter for that block?

Reply:
I get your point, but currently, we don't support filter by `Labels`, how about using kind for that? Please note that I have note to explain this figure right after the image section, but you're right, less confusing is better 👍

Reply:
Looks fine. So let's use Kind in the doc and add a comment in the `chain` section to tell that the label can also be used in the future.

Reply:
Addressed by https://github.com/pipe-cd/pipe/pull/2925/commits/e1f6b885aebe105f5834482d0e7b2464e83a3335, PTAL 👍

Reply:
To be simple instead of having 3 `Application 2` in the second block, only the first two make it look simpler.